app/brave_command_ids.h

```diff
@@ -71,18 +71,19 @@
 #define IDC_BRAVE_BOOKMARK_BAR_ALWAYS 56201
 #define IDC_BRAVE_BOOKMARK_BAR_NTP 56202
 #define IDC_BRAVE_BOOKMARK_BAR_NEVER 56203
+#define IDC_TOGGLE_ALL_BOOKMARKS_BUTTON_VISIBILITY 56204
```

Comment:
This is the new one. others are formatted and the values below this item is increased.

Reply:
I wonder if we really needs to put it here and change everything below.
It seems we currently does not really do grouping and reserve sections for each in this file. I see there is already a IDC_TOGGLE_VERTICAL_TABS_EXPANDED below.
If we want to start having groups, maybe we should do so by reserving some amount of IDs so we don't change everything after each time?

Reply:
FWIW, I think this could also break shortcuts which use the id to store custom shortcuts

Reply:
I put this here to group bookmark related commands. But what @fallaciousreasoning says looks critical. Reverted.

Reply:
Not really in the scope of this PR, but if you don't mind, please start with 56300 so we could have a range for AI_CHAT group.